sunpy/physics/differential_rotation.py

```diff
@@ -447,6 +447,7 @@ def _warp_sun_coordinates(xy, smap, new_observer, **diff_rot_kwargs):
     See :func:`~sunpy.coordinates.transform_with_sun_center`.
     """
     # Suppress NaN warnings in coordinate transforms
+    import warnings  # isort:skip
```

Comment:
I would rather remove the extra pre-commit isort than adding isort:skip for this.